test/Dialect/SV/hw-extract-test-code-error.mlir

```diff
@@ -1,9 +1,20 @@
-// RUN:  circt-opt --sv-extract-test-code %s -verify-diagnostics
-module attributes {firrtl.extract.assert =  #hw.output_file<"dir3/", excludeFromFileList, includeReplicatedOps>, firrtl.extract.assume.bindfile = #hw.output_file<"file4", excludeFromFileList>} {
+// RUN:  circt-opt --sv-extract-test-code --debug-only=extract-test-code %s -verify-diagnostics
```

Comment:
This `RUN` line will not work for release builds, yes? It is reasonable to simply not test this. (I do sincerely appreciate the thoroughness of adding a test for this!)

Reply:
good point, i can just remove this test.

Reply:
FWIW, in the future if we want tests for this, this can be guarded by `asserts` feature-- add a `REQUIRES: asserts` to the lit test (not good idea here since the whole test is covered by that).

Anyway fine to not just making sure we know our options :+1: .

Reply:
Nice, good to know, we can use that in other situations.